cluster/src/dunit/scala/io/snappydata/cluster/ClusterManagerTestBase.scala

```diff
@@ -121,7 +121,8 @@ abstract class ClusterManagerTestBase(s: String)
       }
     }
 
-    Array(vm0, vm1, vm2).foreach(_.invoke(startNode))
+    vm0.invoke(startNode)
+    Array(vm1, vm2).map(_.invokeAsync(startNode)).foreach(_.getResult)
```

Comment:
@sumwale  yes, this will be a good time saver. Just a question - why first vm0 started synchronously while reaming asynchronously ?  why not all three vms started  asynchronously?